android/transit/ui/src/main/java/com/example/transit/ui/TransitFilter.kt

```diff
@@ -0,0 +1,167 @@
+package com.trafi.transit.ui
+
+import androidx.annotation.DrawableRes
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.sizeIn
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.ColorFilter
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.trafi.core.model.FilterItem
+import com.trafi.ui.component.internal.nearby.transit.NearbyTransitFilterItemConstants
+import com.trafi.ui.components.MultiSelectFilter
+import com.trafi.ui.components.SingleSelectFilter
+import com.trafi.ui.theme.MaasTheme
+import com.trafi.ui.theme.currentTheme
+import com.trafi.ui.theme.internal.toColor
+import com.trafi.ui.theme.isRound
+
+@Composable
+private val constants
+    get() = NearbyTransitFilterItemConstants(currentTheme)
+
+@Composable
+fun NearbyTransitFilterItem(
+    modifier: Modifier = Modifier,
```

Comment:
Specify the optional `modifier` parameter as the first *optional* parameter, i.e. after `filterItem` and `isEnabled`